### PR TITLE
MSEdgeRedirect: Add version 0.6.2.0

### LINKED ISSUE #7572: [Request] MSEdgeRedirect
Package Request
Information
Name: MSEdgeRedirect

Description: A Tool to Redirect News, Search, Widgets, Weather and More to Your Default Browser

Homepage: https://github.com/rcmaehl/MSEdgeRedirect

Download link(s): https://github.com/rcmaehl/MSEdgeRedirect/releases/latest/download/MSEdgeRedirect.exe

Some indication of popularity/repute: Github  539 stars

### PR DESCRIPTION
**Pull request by rcmaehl**
Closes #7572 

Generated file based on the file used for #7866 

If anything is wrong let me know :)

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).

### CODE REVIEW

File: bucket/MSEdgeRedirect.json [hunk 1]
```diff
@@ -0,0 +1,34 @@
+{
+    "version": "0.6.2.0",
+    "description": "A Tool to Redirect News, Search, Widgets, Weather and More to Your Default Browser",
+
```
Comment (rashil2000) [inline bucket/MSEdgeRedirect.json:4]:
```suggestion
```

**Comment by rashil2000** [comment]
Add hash to autoupdate: $baseurl/[checksums.sha256](https://github.com/rcmaehl/MSEdgeRedirect/releases/download/0.6.2.0/checksums.sha256)

**Comment by rcmaehl** [comment]
> Add hash to autoupdate: $baseurl/[checksums.sha256](https://github.com/rcmaehl/MSEdgeRedirect/releases/download/0.6.2.0/checksums.sha256)

Will do

**Comment by rashil2000** [comment]
> > Add hash to autoupdate: $baseurl/[checksums.sha256](https://github.com/rcmaehl/MSEdgeRedirect/releases/download/0.6.2.0/checksums.sha256)
> 
> Will do

Could you also update it to new version

**Pull request merged**